packages/pluggableWidgets/video-player-native/package.json

```diff
@@ -20,11 +20,14 @@
   },
   "dependencies": {
     "@mendix/piw-native-utils-internal": "^1.0.0",
-    "react-native-video": "5.0.2"
+    "react-native-system-navigation-bar": "^1.0.2",
+    "react-native-vector-icons": "7.1.0",
+    "react-native-video": "5.2.0"
   },
   "devDependencies": {
     "@mendix/pluggable-widgets-tools": ">=8.9.2",
     "@mendix/piw-utils-internal": "1.0.0",
```

Comment:
I think this should actually move up to dependencies as it's part of production code

Reply:
done